src/mantine-core/src/components/Popper/PopperContainer/PopperContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Portal } from '../../Portal';
+
+export interface PopperContainerProps {
+  /** PopperContainer children, for example, modal or popover */
+  children: React.ReactNode;
+
+  /** Root element z-index property */
+  zIndex?: number;
+
+  /** Root element className */
+  className?: string;
+
+  /** Whether to render the target element in a Portal */
+  withinPortal?: boolean;
+}
+
+export function PopperContainer({
+  children,
+  zIndex = 1,
+  className,
+  withinPortal,
```

Comment:
Let's set this to false by default to be more explicit

Reply:
Done in latest commit